### DISCUSSION TITLE
Release v1.3.2 / 5.58.2

### DISCUSSION THREAD

**Discussion by DavidXanatos** [sandboxie-plus/Sandboxie#2194] [Announcements]
![0 5 5](https://user-images.githubusercontent.com/3890945/105014988-575cb680-5a41-11eb-90d5-d942d28184b2.png)

This build fixes various issues and adds various minor improvements.

#### You can support the project through donations, any help will be greatly appreciated.

[<img src="https://www.paypalobjects.com/en_US/AT/i/btn/btn_donateCC_LG.gif">](https://xanasoft.com/go.php?to=donate)

#### If you have issues with an update installation, just uninstall the previous version keeping the sandboxie.ini and reinstall the new build.

# Changelog

### Added
- added icons to sub tabs in the box options dialog
- recovery and message pop-up menu options are not persisting across UI restarts any more
- added new box color, a white box indicates that its not really a sandbox and is displayed when the user specified OpenFilePath=* or alike

### Changed
- Sandboxie no longer issues message 1301 when forced processes are temporarily disabled
-- the message can be re-enabled with "NotifyForceProcessDisabled=y"
- reworked the "Open COM" checkbox mechanism in the plus UI
-- Now it uses a template and it can also keep COM closed while OpenIpcPath=* is set

### Fixed
- fixed compatibility issue with Proxifier [#2163](https://github.com/sandboxie-plus/Sandboxie/issues/2163)
- fixed encoding issue with Korean translation [#2173](https://github.com/sandboxie-plus/Sandboxie/pull/2173)
- fixed issues with update available message

<hr /><em>This discussion was created from the release <a href='https://github.com/sandboxie-plus/Sandboxie/releases/tag/v1.3.2'>Release v1.3.2 / 5.58.2</a>.</em>

**Comment by BilleBarrett** [comment]
Getting a lot of complaints that systems are rebooting for no reason all
All these systems are running Sandboxie-Plus 1.31, 1.32
All have the latest Windows 11 service packs, etc.
Mine started just rebooting and I uninstalled 1.32 and reinstalled it and everything is running Ok now.

**Reply by bastik-1001** [reply]
Do you also get reports of people that don't use Sandboxie? Or are all machines running it, so there is no way to tell if it is related to Windows 11, itself?

**Comment by BilleBarrett** [comment]
I installed 1.32 on a Surface yesterday and had to remove it this morning

The changed to Sandboxie-plus I made were generic, and web browsers

This morning I did the same to mine, I will send you snapshot if you like, but it is running

DPC was showing when one crashed but I could net read the entire error

 

  <https://go.sentinelone.com/rs/327-MNM-087/images/nurture.bannerv2.1200x200(1).jpg> 
End Point Detection & Response System
Remote Desktop Services
517 Miltwood Dr,  Greensboro, NC 27455-2709
(336) 988-0891    ***@***.***> ***@***.*** 
To subscribe to my FYI emails send an email to
 ***@***.***> ***@***.***
Life is not a journey to the grave with the intention of arriving safely in a pretty and well-preserved body, but rather to skid in broadside, thoroughly used up, totally worn out,
and loudly proclaiming... 
 'Wow! What a ride!’

**Reply by BilleBarrett** [reply]
Today patch my PC updated Sandboxie-Plus 1.3,2, I forgot to turn it off 
But right after that, my system started rebooting, I finally got a Blue screen
I wrote this down ATTEMPTED SWITCH FROM DPC was showing 
I uninstalled Sandboxie-Plus 1.3.2, deleted the configuration file and reinstalled and did my usual setting and so far no problems
BTW this is how I installed Sandboxie-Plus on the other machines that I had to remove, because of rebooting
I am going to screenshot all my settings and then let PatchmytPC do the update and see what is being changed

**Comment by erasmo52** [comment]
Thanks for your job David. The problem In the Sandboxie Classic 5.58.2 it is not possible to execute last Skype release (8.88.0.401). It happens also in the precedent Sandboxie Classic 5.58.1. But the precedent Skype release (8.87.0.406) work fine under both Sandboxie Classic releases.
Thanks for your attention

**Reply by isaak654** [reply]
Logged as #2217

**Comment by BilleBarrett** [comment]
Did I miss something, but I noticed that when I download things the recover windows are no longer open, and everything is still in the secure load folders
I use to be able to right-click on the Sandboxie-Plus Icon and click on recover if it didn't automatically recover to the actual download folder
Did I miss a cleck box, I looked and didn't see one 
Thanks\Bill

**Reply by DavidXanatos** [reply]
is this one checked:
![grafik](https://user-images.githubusercontent.com/3890945/189388191-859a9b0e-42f3-45a7-967d-08dbef507e1e.png)

also have you found out which option caused the BSOD your our systems?

**Comment by BilleBarrett** [comment]
You are right it was not checked, it also a different windows 

Thank you

PS – been busy with a bunch of new clients, not had a chance to look at the problem

I did turn off PatchmyPc entry for Sandboxie-Plus for not



 

  <https://go.sentinelone.com/rs/327-MNM-087/images/nurture.bannerv2.1200x200(1).jpg> 
End Point Detection & Response System
Remote Desktop Services
517 Miltwood Dr,  Greensboro, NC 27455-2709
(336) 988-0891    ***@***.***> ***@***.*** 
To subscribe to my FYI emails send an email to
 ***@***.***> ***@***.***
Life is not a journey to the grave with the intention of arriving safely in a pretty and well-preserved body, but rather to skid in broadside, thoroughly used up, totally worn out,
and loudly proclaiming... 
 'Wow! What a ride!’
